### ISSUE TITLE
CRNN过拟合求解The Problem Of Overfitting

### ISSUE THREAD

**Issue by Boomm-shakalaka** [meijieru/crnn.pytorch#246]
在做CRNN数字辨识时，用mnist的资料集来随机生成不等长数字。总共生成过6w和10w笔资料。损失值到0.00005停止，都发生过拟合现象，验证集准确率很高，但是测试准确率很低，是因为只用mnist来当训练集过于单一吗？
In the process of crnn number identification, the data set of MNIST is used to generate unequal numbers randomly. A total of 6W and 10W data were generated. When the loss value stops at 0.00005, the fitting phenomenon occurs. The accuracy of the verification set is very high, but the accuracy of the test is very low. Is it because only using MNIST as the training set is too single?
